presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -57,6 +61,8 @@
 
     private boolean debug;
 
+    private final ProgressStatsHistory progressStatsHistory = new ProgressStatsHistory(start, new Duration(60, SECONDS));
```

Comment:
Let's make `Duration(60, SECONDS)` to be a separate final variable `statsHistoryInterval`